src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int STARTING_INT = -1;
+    private static final int MAX_TAB_SIZE = 12;
+    private static final int FULL_INT = 11;
+    private int[] numbers = new int[MAX_TAB_SIZE];
 
-    public int total = -1;
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = STARTING_INT;
 
-    public void countIn(int in) {
+    public void countIn(final int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
-    public boolean callCheck() {
-        return total == -1;
+    boolean callCheck() {
+        return total == STARTING_INT;
     }
 
-    public boolean isFull() {
-        return total == 11;
+    boolean isFull() {
+        return total == FULL_INT;
     }
 
-    protected int peekaboo() {
+    int peekaboo() {
         if (callCheck())
-            return -1;
+            return STARTING_INT;
```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)